src/goto-programs/read_bin_goto_object.cpp

```diff
@@ -158,6 +158,8 @@ static bool read_bin_goto_object_v4(
 
   functions.compute_location_numbers();
 
+  POSTCONDITION(functions.function_map.count(goto_functionst::entry_point()));
```

Comment:
Nit pick: Visual Studio warns about this conversion from an integer to a Boolean. So either add `!= 0` or use `functions.function_map.find(...) != functions.function_map.end()`.

Reply:
Added comparison with 0.